DOC.md

```diff
@@ -2809,8 +2809,14 @@ the lazy_load.
   current node as an argnode (will actually only update them if the text in any
   of the argnodes changed).
 
-- `available()`: return a table of all snippets defined for the current
-  filetypes(s) (`{ft1={snip1, snip2}, ft2={snip3, snip4}}`).
+- `available(snip_info)`: return a table of all snippets defined for the
+  current filetypes(s) (`{ft1={snip1, snip2}, ft2={snip3, snip4}}`).
+  The structure of the snippet is defined by `snip_info` which is a function
+  (`snip_info(snip)`) that takes in a snippet (`snip`) and uses it to define
+  the desired table.
```

Comment:
Maybe `... , finds the desired information on it, and returns it`

Reply:
Like this you mean? ```(`snip_info(snip)`) that takes in a snippet (`snip`), finds the desired
  information on it, and returns it.``` ?

Reply:
Yeah, something like it👍
I'd like this passage to have more context, `uses it to define the desired table` sounds a bit too general imo

Reply:
Done! https://github.com/L3MON4D3/LuaSnip/pull/652/commits/5eace87c76cd733bc8f0be71e17098a88e408d11